docs/integrations/databases/opentelemetry/cassandra-opentelemetry.md

```diff
@@ -0,0 +1,171 @@
+---
+id: cassandra-opentelemetry
+title: Cassandra - OpenTelemetry Collector
+sidebar_label: Cassandra - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for Cassandra.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/databases/cassandra.png')} alt="Thumbnail icon" width="75"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The [Cassandra](https://cassandra.apache.org/_/cassandra-basics.html) - OpenTelemetry app is a log based app that helps you monitor the availability, performance, health, and resource utilization of your Cassandra clusters. Preconfigured dashboards provide insight into resource utilization, cache/Gossip/Memtable statistics and Error and warnings. Cassandra logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+The app supports Logs from the open-source version of Cassandra. The App is tested on the 3.11.10 version of Cassandra.
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Cassandra-OpenTelemetry/Schematics.png')} alt="Schematics" />
```

Comment:
Image link is broken. When I go to the URL address in a browser, I get an "Access Denied" error.